app/assets/stylesheets/bootstrap_and_overrides.css

```diff
@@ -62,24 +68,16 @@
     padding-top: 50px;
   }
 
-  body
-  {
+  body {
     font-family: 'Hind', sans-serif;
     font-size: 16px;
-    padding-top: 80px;
     color: #555;
     background-image: url('http://i.imgur.com/WtYG1kl.png');
     background-attachment: fixed;
-    background-repeat: no-repeat;
```

Comment:
@GSmes Why did this line need to be removed? Not at all saying it's the wrong move, just curious since I wasn't there for some of the hamburger discussion.